### PR TITLE
my-bike

### PR DESCRIPTION
**Pull request by SergeyShapovalenko**
[DEMO LINK](https://SergeyShapovalenko.github.io/layout_miami/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1 +1,10 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu')
+  } else {
+    document.body.classList.remove('page__body--with-menu')
+  }
+
+})
```
Comment (SerhiiKirik) [inline src/scripts/main.js:10]: add empty line
```suggestion
})

```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,407 @@
...
+              alt="MyBike">
+            </a>
+            <div class="icons">
+              <a 
+              href="tel:+1 234 555-55-55" class="icons icons--call"
+              >
+                <div class="icons__tooltip">
+                +1 234 555-55-55
+                </div>
+              </a>
```
Comment (SerhiiKirik) [inline src/index.html:31]:
```suggestion
              <img 
                class="logo__image"
                src="images/logo.svg" 
                alt="MyBike"
              >
            </a>
            
            <div class="icons">
              <a 
                href="tel:+1 234 555-55-55" 
                class="icons icons--call"
              >
                <div class="icons__tooltip">
                +1 234 555-55-55
                </div>
              </a>
```

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,407 @@
...
+          <a 
+            href="tel:+1 234 5555-55-55" 
+            class="menu__call menu__call--phone"
+            >
+            +1 234 5555-55-55
+          </a>
+          <a 
+            href="tel:+1 234 5555-55-55" 
+            class="menu__call menu__call--action"
+            >
```
Comment (SerhiiKirik) [inline src/index.html:79]: fix it in all cases
```suggestion
          <a 
            href="tel:+1 234 5555-55-55" 
            class="menu__call menu__call--phone"
          >
            +1 234 5555-55-55
          </a>
          
          <a 
            href="tel:+1 234 5555-55-55" 
            class="menu__call menu__call--action"
          >
```

File: src/styles/blocks/_bike-card.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
+.bike-card {
+  
+  &__photo {
+    &:hover {
+      @include bike-animation;
+    }
+
+
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/_bike-card.scss:9]:
```suggestion
```

File: src/styles/blocks/_button.scss [hunk 5]
```diff
@@ -0,0 +1,42 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  
+  box-sizing: border-box;
```
Comment (SerhiiKirik) [inline src/styles/blocks/_button.scss:6]: put it in `*` selector into normalized file
```suggestion
```

File: src/styles/blocks/_button.scss [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  width: 100%;
+  height: 56px;
+
+  text-decoration: none;
+  color: $basic-color;
+  background: #fff;
```
Comment (SerhiiKirik) [inline src/styles/blocks/_button.scss:12]: don't forget to create variables for all colors
```suggestion
  background: $c-white;
```

File: src/styles/blocks/_contact-us.scss [hunk 7]
```diff
@@ -0,0 +1,31 @@
+.contact-us {  
+  &__info {
+    &:not(:last-child) {
+      margin: 0 0 24px;
+    }
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/_contact-us.scss:6]:
```suggestion
```

File: src/styles/blocks/_contact-us.scss [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    &:not(:last-child) {
+      margin: 0 0 24px;
+    }
+   
+  }
+
+  &__info-title {
+    margin: 0 0 8px;
+    
+    font-weight: 400;
```
Comment (SerhiiKirik) [inline src/styles/blocks/_contact-us.scss:12]: it's a default value
remove in all cases
```suggestion
```

File: src/styles/blocks/_contact-us.scss [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+    color:#fff;
+  }
+
+  &__field {
+    margin: 0 0 16px;
+    &--last {
+      margin-bottom: 32px;
+    }
+  }
+}
```
Comment (SerhiiKirik) [inline src/styles/blocks/_contact-us.scss:31]: add empty line
```suggestion
}

```

File: src/styles/blocks/_contact-us.scss [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 140%;
+    text-decoration: none;
+    color:#fff;
+  }
+
+  &__field {
+    margin: 0 0 16px;
+    &--last {
```
Comment (SerhiiKirik) [inline src/styles/blocks/_contact-us.scss:27]:
```suggestion
    margin: 0 0 16px;
    
    &--last {
```

File: src/styles/blocks/_form-field.scss [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+  }
+
+  &--textarea {
+    font-family: inherit;
+    height: 148px;
+    resize: none;
+    border-radius: 16px;
+  }
+
+}
```
Comment (SerhiiKirik) [inline src/styles/blocks/_form-field.scss:35]:
```suggestion
}

```

File: src/styles/blocks/_grid.scss [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+  @include onDesktop {
+    grid-template-columns: repeat(12, 1fr);
+    column-gap: 24px;
+  }
+
+  &--mobile {
+    display: block;
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/_grid.scss:18]:
```suggestion
```

File: src/styles/blocks/_menu.scss [hunk 13]
```diff
@@ -0,0 +1,93 @@
+.menu {
+  background-color: #c2c2c2;
+  transition: $transition;
+  &__content {
+    padding: $header-top-padding 0;
+    height: 100vh;
+
+    @include onTablet() {
+      //height: 630px;
```
Comment (SerhiiKirik) [inline src/styles/blocks/_menu.scss:9]: remove the comments 
in all cases
```suggestion
```

File: src/styles/blocks/_menu.scss [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+    border-bottom: 1px solid transparent;
+    color: $basic-color;
+    font-weight: 600;
+    font-size: 22px;
+    line-height: 22px;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    transition: border-color 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/_menu.scss:64]: create a variable for $transitionDuration
```suggestion
    transition: border-color $transitionDuration;
```

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Great 🔥 
Let's improve your code & fix problems 🧑‍💻 
I make a video for you - [video](https://vimeo.com/805189329)

link - https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

File: src/styles/blocks/_menu.scss (hunk 14 above)
Comment (SergeyShapovalenko) [reply src/styles/blocks/_menu.scss:64]: I will fix it, thank you very much for your work!)

File: src/index.html [hunk 15]
```diff
@@ -1,13 +1,423 @@
...
+              </a>
+  
+              <a href="#menu" class="icons icons--menu-opener"></a>
+            </div>
+          </div>
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+    <nav class="page__menu menu" id="menu">
```
Comment (SerhiiKirik) [inline src/index.html:41]:
```suggestion
          <div class="header__top">
            <a href="#home" class="logo">
              <img 
                class="logo__image"
                src="images/logo.svg" 
                alt="MyBike">
            </a>
            
            <div class="icons">
              <a 
                href="tel:+1 234 555-55-55" 
                class="icons icons--call"
              >
                <div class="icons__tooltip">
                +1 234 555-55-55
                </div>
              </a>
  
              <a href="#menu" class="icons icons--menu-opener"></a>
            </div>
          </div>
          
          <h1 class="header__title">Take the Streets</h1>
        </div>
      </div>
    </header>
    
    <nav class="page__menu menu" id="menu">
```

File: src/index.html [hunk 16]
```diff
@@ -1,13 +1,423 @@
...
+            href="tel:+1 234 5555-55-55" 
+            class="menu__call menu__call--phone"
+            >
+            +1 234 5555-55-55
+          </a>
+          
+          <a 
+            href="tel:+1 234 5555-55-55" 
+            class="menu__call menu__call--action"
+            >
```
Comment (SerhiiKirik) [inline src/index.html:86]: Pls, pay attention
Fix it in all cases, in all file
```suggestion
          >
            +1 234 5555-55-55
          </a>
          
          <a 
            href="tel:+1 234 5555-55-55" 
            class="menu__call menu__call--action"
          >
```

File: src/index.html [hunk 17]
```diff
@@ -1,13 +1,423 @@
...
+          <div class="grid grid--mobile">
+            <div class="grid__item grid__item--onDesktop-1-5 grid__item--onTablet-1-3">
+              <form action="#" method="get">
+                <input 
+                required
+                name="name" 
+                type="text"
+                class="form-field contact-us__field"
+                placeholder="Name"
+                >
```
Comment (SerhiiKirik) [inline src/index.html:347]:
```suggestion
                <input 
                  required
                  name="name" 
                  type="text"
                  class="form-field contact-us__field"
                  placeholder="Name"
                >
```

File: src/styles/blocks/_bike-card.scss [hunk 18]
```diff
@@ -0,0 +1,36 @@
+.bike-card {
+
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/_bike-card.scss:3]:
```suggestion
```

File: src/styles/blocks/_header.scss [hunk 19]
```diff
@@ -0,0 +1,50 @@
+.header {
+  background: url(../images/header.png) right/cover no-repeat;
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/_header.scss:3]:
```suggestion
```

File: src/styles/blocks/_place.scss [hunk 20]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  &__photo {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+
+    object-fit: cover;
+    object-position: center;
+    transition: transform 1s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/_place.scss:38]: use have var for this
```suggestion
    transition: transform $transition-duration;
```

File: src/styles/blocks/_page.scss [hunk 21]
```diff
@@ -0,0 +1,85 @@
...
+    @include onTablet {
+      margin-bottom: 120px;
+    }
+  }
+
+  &__section-title {
+    padding: 0 0 48px;
+    margin: 0;
+
+    font-style: normal;
```
Comment (SerhiiKirik) [inline src/styles/blocks/_page.scss:52]: it's a default value for this style
```suggestion
```

File: src/styles/blocks/_menu.scss [hunk 22]
```diff
@@ -0,0 +1,92 @@
+.menu {
+  background-color: $silver;
+  transition: $transition-duration;
+
+  z-index: 1;
+  height: 100vh;
+  overflow-y: auto;
+  &__content {
+    padding: $header-top-padding 0;
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/_menu.scss:10]:
```suggestion
```

File: src/styles/blocks/_menu.scss [hunk 23]
```diff
@@ -0,0 +1,92 @@
+.menu {
+  background-color: $silver;
+  transition: $transition-duration;
+
+  z-index: 1;
+  height: 100vh;
+  overflow-y: auto;
+  &__content {
```
Comment (SerhiiKirik) [inline src/styles/blocks/_menu.scss:8]:
```suggestion
  overflow-y: auto;
  
  &__content {
```

File: src/styles/blocks/_menu.scss [hunk 24]
```diff
@@ -0,0 +1,92 @@
+.menu {
+  background-color: $silver;
+  transition: $transition-duration;
```
Comment (SerhiiKirik) [inline src/styles/blocks/_menu.scss:3]: use value for this transition

**Review by SerhiiKirik — APPROVED** [review]
LGTM 🔥

**Pull request closed without merging**